robot/Cumulus/resources/NPSP.robot

```diff
@@ -331,10 +316,24 @@ Enable Customizable Rollups
     Go To Page                                Custom          NPSP_Settings
     Open Main Menu                            Donations
     Click Link With Text                      Customizable Rollups
-    Enable Customizable Rollups If Not Enabled         
+    Enable Customizable Rollups If Not Enabled
 Enable Advanced Mapping
-    [Documentation]    This keyword enables advanced mapping if not already enabled.  
+    [Documentation]    This keyword enables advanced mapping if not already enabled.
     Go To Page                              Custom          NPSP_Settings
     Open Main Menu                          System Tools
-    Click Link With Text                    Advanced Mapping for Data Import & Gift Entry   
-    Enable Advanced Mapping If Not Enabled 
+    Click Link With Text                    Advanced Mapping for Data Import & Gift Entry
+    Enable Advanced Mapping If Not Enabled
+
+    Enable Customizable Rollups If Not Enabled
```

Comment:
please remove this line from Enable Advanced Mapping keyword. Looks like u missed it while resolving conflicts